datacenter/dtr/2.2/guides/admin/monitor-and-troubleshoot/troubleshoot-with-logs.md

```diff
@@ -20,45 +20,20 @@ You can also use any images that contain `sh` and `ping` for this test.
 
 If the second command succeeds, overlay networking is working.
 
-## DTR doesn't come up after a Docker restart
+## Accessing rethinkdb directly
 
-This is a known issue with Docker restart policies when DTR is running on the same
-machine as a UCP controller. If this happens, you can simply restart the DTR replica
-from the UCP UI under "Applications". The best workaround right now is to not run
-DTR on the same node as a UCP controller.
+To perform operations against rethinkdb directly, one may use the following
```

Comment:
If we were talking face to face you'd not say "one may use the following". 🎩🥃
Maybe

You can connect directly to the rethinkdb data store internal to DTR for troubleshooting. Run this command on a host that's running a DTR replica to start a web UI that allows running queries on the rethinkdb data store.